artview/components/plot_radar.py

```diff
@@ -602,11 +602,16 @@ def toolZoomPanCmd(self):
 
     def toolValueClickCmd(self):
         '''Creates and connects to Point-and-click value retrieval'''
-        from .tools import ValueClick
-        self.tools['valueclick'] = ValueClick(
-            self.Vradar, self.Vtilt, self.Vfield,
-            self.units, self.ax, self.statusbar, parent=self.parent)
-        self.tools['valueclick'].connect()
+        if self.plot_type == 'radarPpi':
+            from .tools import ValueClick
+            self.tools['valueclick'] = ValueClick(
+                self.Vradar, self.Vtilt, self.Vfield,
+                self.units, self.ax, self.statusbar, parent=self.parent)
+            self.tools['valueclick'].connect()
+        else:
+            import warnings
+            warnings.warn("valueclick not implemented for airborne or rhi",
+                          NotImplementedError)
```

Comment:
I don't recall ValueClick not working with RHI/Airborne. Maybe I never tested....